RELEASE-NOTES.txt

```diff
@@ -1,6 +1,12 @@
 Unreleased
 ==========
 
+1.5.5
+=====
+* Improved the pull and push functionality to allow selective sync
+* Misc UI fixes #1538, #1533, #1534, #1494
```

Comment:
Would it make sense to move 1538 to a separate line as "Added token revoke call during logout to improve security"?